components/ingest-service/backend/elastic/mappings/node_state.go

```diff
@@ -262,10 +280,18 @@ var nodeProps = `
 		"type": "ip"
 	},
 	"source_fqdn": {
-		"type": "keyword"
+		"type": "keyword",
+		"normalizer": "case_insensitive",
+		"fields": {
+			"engram" : {
+				"type": "text",
+				"analyzer": "autocomplete"
+			}
+		}
```

Comment:
Adding the `source_fqdn` and `organization_name` field to allow wildcard filters.